### DISCUSSION TITLE
How to update Cloudflared out of band?

### DISCUSSION THREAD

**Discussion by fokcuk** [homeassistant-apps/app-cloudflared#236] [Q&A] [answered]
Getting reminders that cloudflared needs an update, but no new release
How do I update it out of band?

**Comment by brenner-tobias** [comment, marked as answer]
You cannot. We do usually release the update very quickly after Cloudflare releases it (which I just did). We do check for any problems that might arise within the add-on, so you really don't have to worry about it too much.
